Rancher/docs/rdt/readme.md

```diff
@@ -35,8 +40,10 @@ This can be done several ways however for this exercise we will be using VSCode.
 - [ ] Brief repo overview
 	- [ ] Navigate to the location in the repo used in this exercise
 
+[//]: # (Do we expect participants to understand what k3s is? If not, maybe short synposis of what the k3s distribution is could be useful)
```

Comment:
This may be useful but in reality K3s is what Rancher Desktop installs under the hood so this reference would be purely background information and have no functional effect on the outcome.